erc20-connector/contracts/BridgeTokenFactory.sol

```diff
@@ -56,7 +56,7 @@ contract BridgeTokenFactory is ERC20Locker {
 
     function newBridgeToken(string calldata nearTokenId) external returns (BridgeToken) {
         require(!_isBridgeToken[_nearToEthToken[nearTokenId]], "ERR_BRIDGE_TOKEN_EXISTS");
-        BridgeToken btoken = new BridgeToken();
+        BridgeToken btoken = new BridgeToken("Rainbow", "RB");
```

Comment:
Do we want to have this default values or do we prefer using something like: `f"rainbow-{nearTokenId}"`